### ISSUE TITLE
HexKinetics' Sniper's Distillation Hits Anti-Raycast Blocks

### ISSUE THREAD

**Issue by OrionOth** [beholderface/oneironaut#32]
HexKinetics' Sniper's Distillation ([OpPixelRaycast](https://github.com/Sonunte/HexKinetics/blob/387f5878c9857216dc0d952a49e992709ca6ef8c/Common/src/main/java/net/sonunte/hexkinetics/common/casting/actions/OpPixelRaycast.kt)) returns an accurate location on the Anti-Raycast blocks (glass/slate).

This is also a problem with HexGloop's Hexxed Glass.

Versions (truncated)
- Minecraft 1.19.2
- Quilt Loader 0.26.3
- Hex Casting 0.10.3
- Oneironaut 0.3.4
- HexKinetics 0.7.3
- HexGloop 0.2.0

**Comment by beholderface** [comment]
Yeah I've known this was probably the case since I added the blocks (pretty sure sniper's also ignores stealth shroud), but I'm not sure how to make the mixin I'd probably need only try to load when hexkinetics is installed.

Also I don't really want to acknowledge hexkinetics in the first place, since it's so damn jank.

**Comment by walksanatora** [comment]
@Pseudo a mixin into hex kinetics. thatway it will silently fail if it cannot inject.
